Source/WebCore/platform/graphics/gstreamer/GStreamerVideoFrameConverter.cpp

```diff
@@ -124,8 +166,14 @@ GRefPtr<GstSample> GStreamerVideoFrameConverter::convert(const GRefPtr<GstSample
     return convertedSample;
 }
 
+void GStreamerVideoFrameConverter::releaseUnusedPipelinesTimerFired()
+{
+    m_systemMemoryPipeline = nullptr;
```

Comment:
Is this really sufficient?

Usually to cleanup a pipeline we set its state to NULL first, but maybe here it's not needed since `convert()` does it already.

When re-creating the pipeline though, I'm not sure what happens with m_src and m_sink... are they re-parented?

Reply:
I had the same doubts, so I tried with a shorter cleanup time and the pipeline was correctly destroyed/created. In any case I'm not yet convinced that a singleton is the best approach... But I want a simple patch for the stable branch.